webapp/channels/src/components/post/post_component.tsx

```diff
@@ -513,7 +513,7 @@ const PostComponent = (props: Props): JSX.Element => {
             className={props.location === 'SEARCH' ? 'search-item__container' : undefined}
             data-testid={props.location === 'SEARCH' ? 'search-item-container' : undefined}
         >
-            {(isSearchResultItem || (props.location !== Locations.CENTER && props.isPinnedPosts) || props.isFlagged) && <DateSeparator date={currentPostDay}/>}
+            {(isSearchResultItem || (props.location !== Locations.CENTER && props.isPinnedPosts)) && <DateSeparator date={currentPostDay}/>}
```

Comment:
Doesn't this hide the DateSeparator in Saved Posts?

![CleanShot 2023-06-08 at 09 06 14@2x](https://github.com/mattermost/mattermost/assets/11724372/15a9bce8-10ee-4d4d-b33b-cbd54d867cfc)

Reply:
Or will this be filled via https://github.com/mattermost/mattermost/pull/23525/files/cdbbe1f7b4ae6b82b770c8eb0cd6e1984d172540..7350ec1d1e46a4e0ee723215cbe7b4ca8ac990ea#diff-1470e9f815f1eb508bd245995d0e123659b7cee8b838943567c8c066494e1d78R513 ?

Reply:
Ahh, yeah that's not intended. I might need Harrison's changes after all. Will fix.